hw4_epam_lab/src/test/java/com/atw/AtwTest.java

```diff
@@ -0,0 +1,43 @@
+package com.atw;
+
+import com.atw.card.Card;
+import com.atw.card.DebetCard;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AtwTest {
+
+    @Test
+    void withdrawAtw_ShowNewBalanceCard_WhenWithdraw() {
+        Card card = new DebetCard(50, "Bob");
+        Atm cardAtm = new Atm(card);
+        cardAtm.withdrawAtm(30);
+        assertEquals(card.getAccountBalance(), 20, 0.0000001);
+    }
+
+    @Test
+    void cardBalance_ShowBalanceCard() {
+        Card card = new DebetCard(50, "Bob");
+        Atm cardAtm = new Atm(card);
+        cardAtm.cardBalance();
```

Comment:
Does it make any sense?
It has to be like this:
`assertEquals(cardAtm.cardBalance(), 50, 0.0000001);`
And line 23 has to be removed

Reply:
Done